nixos/modules/services/hardware/monado.nix

```diff
@@ -16,6 +16,7 @@ let
 
   cfg = config.services.monado;
 
+  package = cfg.package.override { withUltraleap = cfg.enableUltraleap; };
```

Comment:
It could be better to use `apply` on the package option itself rather than override it in in a local scope like this -- as the former would allow users to inspect the package actually being used